exe/stepmod-annotate

```diff
@@ -0,0 +1,39 @@
is", "aic_draughting_annotation"=>"draughting_annotation", "aic_geometric_tolerances"=>"geometric_tolerances", "aic_faceted_brep"=>"faceted_boundary_representation", "shape_aspect_definition_schema"=>"shape_variation_tolerances", "shape_dimension_schema"=>"shape_variation_tolerances", "shape_tolerance_schema"=>"shape_variation_tolerances", "kinematic_property_schema"=>"kinematics", "kinematic_topology_schema"=>"kinematics", "kinematic_structure_schema"=>"kinematics", "kinematic_state_schema"=>"kinematics", "kinematic_motion_representation_schema"=>"kinematics", "kinematic_analysis_control_and_result_schema"=>"kinematics", "mesh_connectivity_schema"=>"mesh-based_topology", "mesh_function_schema"=>"mesh-based_topology", "mesh_topology_schema"=>"mesh-based_topology", "expression_extensions_schema"=>"expression_extensions", "logical_expressions_schema"=>"expression_extensions", "aic_mechanical_design_geometric_presentation"=>"mechanical_design_geometric_presentation", "aic_curve_swept_solid"=>"curve_swept_solid", "geometry_schema"=>"geometric_and_topological_representation", "topology_schema"=>"geometric_and_topological_representation", "geometric_model_schema"=>"geometric_and_topological_representation", "scan_data_3d_shape_model_schema"=>"geometric_and_topological_representation", "product_data_equivalence_definition_schema"=>"equivalence_validation_of_product_data", "product_data_equivalence_criteria_schema"=>"equivalence_validation_of_product_data", "product_data_equivalence_inspection_result_schema"=>"equivalence_validation_of_product_data", "annotated_3d_model_equivalence_criteria_schema"=>"equivalence_validation_of_product_data", "annotated_3d_model_equivalence_inspection_result_schema"=>"equivalence_validation_of_product_data", "aic_draughting_elements"=>"draughting_elements", "topology_based_identification_schema"=>"persistent_identification_of_elements_in_procedural_shape_modelling", "aic_topologically_bounded_surface"=>"topologically_bounded_surface", "mathematical_functions_schema"=>"mathematical_constructs", "aic_geometrically_bounded_surface"=>"geometrically_bounded_surface", "classification_schema"=>"classification_and_set_theory", "set_theory_schema"=>"classification_and_set_theory", "method_definition_schema"=>"process_structure_and_properties", "process_property_representation_schema"=>"process_structure_and_properties", "process_property_schema"=>"process_structure_and_properties", "assembly_constraint_schema"=>"kinematic_and_geometric_constraints_for_assembly_models", "assembly_feature_relationship_schema"=>"kinematic_and_geometric_constraints_for_assembly_models", "risk_schema"=>"risk", "product_structure_schema"=>"product_structure_configuration", "product_concept_schema"=>"product_structure_configuration", "configuration_management_schema"=>"product_structure_configuration", "aic_machining_feature"=>"machining_features", "procedural_sketch_schema"=>"modelling_commands_for_the_exchange_of_procedurally_represented_2d_cad_models"}
```

Comment:
Can we make this dynamic?

Reply:
I had it dynamic before, but then it wasn't cached across runs. Do you think about storing it into a temporary file?

Reply:
Sure, that works. We just don't want to hard code these in case files change.

Reply:
Changed:
- **stepmod-annotate** - requires an additional argument with path to a cache file
- **stepmod-annotate-all** - creates a temporary cache file